### PR TITLE
Improve BackupSheet + core SlackSheet component

### PR DESCRIPTION
**Pull request by mikedemarais**
Fixes RAI-782 https://linear.app/rainbow/issue/RAI-782/on-open-keyboard-comes-up-just-barely-covering-the-confirm-button-im

### CODE REVIEW

File: src/screens/BackupSheet.js [hunk 1]
```diff
@@ -133,20 +112,10 @@ const BackupSheet = () => {
     onSuccess,
   ]);
 
-  const onManualBackup = useCallback(() => {
-    switchSheetContentTransitionRef.current?.animateNextTransition();
-    setStep(WalletBackupStepTypes.manual);
-    setOptions({
```
Comment (osdnk) [inline src/screens/BackupSheet.js:139]: We're no longer using this logic, correct?

Comment (mikedemarais) [reply src/screens/BackupSheet.js:139]: correct! i refactored how we handle sheet's height for each step.

**Review by brunobar79 — CHANGES_REQUESTED** [review]
Code looks good. Found some bugs and posted the videos on slack

File: src/components/backup/BackupConfirmPasswordStep.js [hunk 2]
```diff
@@ -1,143 +1,89 @@
...
+  ${padding(0, 50)};
 `;
 
-const InputsWrapper = styled(View)`
-  align-items: center;
-  height: 120;
+const InputsWrapper = styled(Column).attrs({
+  align: 'center',
+})`
+  flex: 1;
```
[the remark this answers is not in the document]
Comment (mikedemarais) [reply src/components/backup/BackupConfirmPasswordStep.js:39]: no advantage, tbh i sorta just pick whichever syntax seems the most concise.

i got rid of these InputWrapper's tho and replaced them with using unnamed Column's directly in the render function

**Review by brunobar79 — APPROVED** [review]
LGTM!

**Pull request merged**